scripts/update_sql.sh

```diff
@@ -2,13 +2,15 @@
 
 cd $(dirname $0)/../mbdata
 
+TARGET_BRANCH="${1:-master}"
```

Comment:
About your second commit

> Allow specifying MBS branch to pull sql

Thank you for this helpful improvement! It will save us from kludges such as https://github.com/metabrainz/mbdata/commit/e54b7167681c893f1f2127b4d480278d562a6f39.

To make it proper, we should even prevent involuntarily querying the `master` branch, to avoid any mismatches.

Since GiHub allows targeting tags (and commits) too, I would suggest to target the latest schema change release tag instead of the `master` branch by default, as follows (untested):

```suggestion
GIT_TARGET="${1:-v-2024-05-13-schema-change}"
```

It also requires updating the development docs.

Reply:
Sounds sensible, will amend.